.github/workflows/pr.yml

```diff
@@ -87,6 +95,12 @@ jobs:
         tag:
           - 'basic || basic-raft || (advanced && raft) || networks/typical::raft'
           - 'basic || basic-istanbul || (advanced && istanbul && !block-heights) || networks/typical::istanbul'
+          - 'validator-management && networks/template::istanbul-3plus1'
```

Comment:
This looks good for PRs.

For `master` we should be running all tests.

Reply:
same as https://github.com/ConsenSys/quorum-acceptance-tests/blob/master/.github/workflows/run.yml#L97

Reply:
solved